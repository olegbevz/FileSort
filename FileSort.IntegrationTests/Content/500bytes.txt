52685. the best
335583. Cherry Apple is
223399. is the Cherry best Apple Apple is
423778. Banana Banana Cherry Something Banana Something
355795. Apple Something
337629. Apple best
958936. Banana Apple the
858830. Banana the best Banana Something Apple
189992. Cherry best
852069. Apple is Banana best
682673. Something best Something best
353740. Banana Banana best Cherry is Banana best best
564947. best the Cherry is Cherry Apple Something Apple the
416924. the Apple Cherry Cherry
