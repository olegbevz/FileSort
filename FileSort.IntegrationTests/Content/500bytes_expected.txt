355795. Apple Something
337629. Apple best
852069. Apple is Banana best
958936. Banana Apple the
423778. Banana Banana Cherry Something Banana Something
353740. Banana Banana best Cherry is Banana best best
858830. Banana the best Banana Something Apple
335583. Cherry Apple is
189992. Cherry best
682673. Something best Something best
564947. best the Cherry is Cherry Apple Something Apple the
223399. is the Cherry best Apple Apple is
416924. the Apple Cherry Cherry
52685. the best
